packages/react-accordion/Spec.md

```diff
@@ -34,104 +34,107 @@ Sample usages will be given in the following section of this document [Sample co
 The root level component serves context and common API between all children.
 
 ```ts
-export interface AccordionProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
-  /**
-   * Indicates if keyboard navigation is available
-   */
-  navigable?: boolean;
-  /**
-   * Indicates if Accordion support multiple Panels opened at the same time
-   */
-  multiple?: boolean;
-  /**
-   * Indicates if Accordion support multiple Panels closed at the same time
-   */
-  collapsible?: boolean;
-  /**
-   * value indicating the items that are opened
-   * If used, the component will be in controlled mode
-   */
-  openItems?: AccordionItemValue | AccordionItemValue[];
-  /**
-   * Index indicating the panels that are opened
-   */
-  defaultOpenItems?: AccordionItemValue | AccordionItemValue[];
-  /**
-   * Size of spacing in the heading
-   */
-  size?: 'small' | 'medium' | 'large' | 'extra-large';
-  /**
-   * The component to be used as button in the heading
-   */
-  button?: ShorthandProps<React.HTMLAttributes<HTMLElement>>;
-  /**
-   * Expand icon slot rendered before (or after) children content in heading
-   */
-  expandIcon?: ShorthandProps<AccordionHeaderExpandIconProps>;
-  /**
-   * The position of the expand  icon slot in heading
-   */
-  expandIconPosition?: 'start' | 'end';
-  /**
-   * Extra icon slot rendered before children content in heading
-   */
-  icon?: ShorthandProps<AccordionHeaderIconProps>;
-  /**
-   * Indicates if the AccordionHeader should be inline-block
-   */
-  inline?: boolean;
-  onToggle?(event: AccordionToggleEvent, data: AccordionToggleData): void;
-}
+export type AccordionProps = ComponentProps &
```

Comment:
Rather than continue to keep this in sync with the actual props, should we just update this to point to the react-*.api.md or the actual types file (tho given the actual typoes are a little harder to read)

Reply:
Agreed, after implementation is done I don't think there's much sense to having the types fully duplicated in the spec